templates/part.board.sidebarView.php

```diff
@@ -118,4 +120,31 @@
             </ul>
 
     </div>
+
+    <div id="board-detail-deleted-stacks" class="tab deletedStacksTabView" ng-if="params.tab==2">
+    	<ul>
+    	   <li ng-repeat="deletedStack in stackservice.deleted">
+    			<dl>
+    			  <dt>Title</dt>
+    			  <dd>{{deletedStack.title}}<dd>
+    			</dl>
+        	<a ng-click="stackUndoDelete(deletedStack)"><span class="icon icon-undo"></span><br /><span><?php p($l->t('Undo delete')); ?></span></a>
+    	   </li>
+    	</ul>
+    </div>
+
+    <div id="board-detail-deleted-cards" class="tab deletedCardsTabView" ng-if="params.tab==3">
+    	<ul>
+    	   <li ng-repeat="deletedCard in cardservice.deleted">
+    			<dl>
+    			  <dt>Title</dt>
+    			  <dd>{{deletedCard.title}}<dd>
+    			  <dt>Stack</dt>
+    			  <dd>{{stackservice.data[deletedCard.stackId].title}}</dd>
```

Comment:
Needs a fix for deleted stacks (not showing when stack is deleted)

- [ ] Add a method to stackservice which returns the title for both deleted and live stacks
- [ ] use it here

Reply:
We cannot restore them if the stack is deleted so it should be fine to just hide them from the UI, right?

Reply:
we can restore them, the user is asked if he wants to restore the stack as well. if they confirm, stack is restored as well, if not, nothing is done, the card is left deleted.